### PR TITLE
Style corrections

### PR DESCRIPTION
**Pull request by mlunoe**
Adding `"use strict";`, correcting indentation, naming modules (`var App = ...` and `module.exports = App;`), adding space to anonymous functions and if-statements (`function ()_{..}` and `if_(){...}`)

### CODE REVIEW

**Comment by mlunoe** [comment]
This should be that final changes for getting the build system in.

File: src/main/resources/assets/Brocfile.js [hunk 1]
```diff
@@ -46,15 +48,8 @@ var tasks = {
...
-          case "production":
-            // TODO(mlunoe) disabled for production
-            break;
-          default:
-            // use pretty colors in test and development mode
-            console.log(chalk.red(message) + "\n");
-            break;
-        }
+        // use pretty colors in test and development mode
+        console.log(chalk.red(message) + "\n");
```
Comment (pyronicide) [inline src/main/resources/assets/Brocfile.js]: Why aren't we still disabling logging for production?

File: src/main/resources/assets/js/components/AppComponent.jsx [hunk 2]
```diff
@@ -1,47 +1,51 @@
 /** @jsx React.DOM */
 
+"use strict";
 
 var React = require("react/addons");
 
-module.exports = React.createClass({
```
Comment (pyronicide) [inline src/main/resources/assets/js/components/AppComponent.jsx]: I kinda prefer this style to the new one. It forces you to only have one component/file and removes any possible repetition.

File: src/main/resources/assets/Brocfile.js (hunk 1 above)
Comment (mlunoe) [inline src/main/resources/assets/Brocfile.js]: Yes, good question. I'll undo the change

File: src/main/resources/assets/js/components/AppComponent.jsx (hunk 2 above)
Comment (mlunoe) [inline src/main/resources/assets/js/components/AppComponent.jsx]: meaning `module.exports = ` instead of `var LOL = ...` and `module.exports = LOL;`?

**Comment by pyronicide** [comment]
:shipit:

**Pull request merged**